### ISSUE TITLE
How can I only evaluate based on the ground-truth bounding box?

### ISSUE THREAD

**Issue by sunformoon** [MIC-DKFZ/medicaldetectiontoolkit#128]
Thanks for the repo!
I am wondering how can I get the evaluation performance (mainly the classification results) given the ground-truth bounding box?

**Comment by sunformoon** [comment]
To clarify, my major task is the classification task, so I would like to obtain the classification results based on the given ground-truth bounding boxes. However, I trained with the detection + classification + segmentation label, in which case I have much more voxel-level annotation to prevent the model from overfitting since I don't have many CT cases.

@pfjaeger Any comments would be greatly appreciated!
